src/Android/Effects/NoEmojiKeyboardEffect.cs

```diff
@@ -1,17 +1,17 @@
-﻿using Bit.Droid.Effects;
+﻿using Android.Widget;
+using Bit.Droid.Effects;
 using Xamarin.Forms;
 using Xamarin.Forms.Platform.Android;
 
-[assembly: ExportEffect(typeof(NoEmojiKeyboardEffect), "NoEmojiKeyboardEffect")]
+[assembly: ExportEffect(typeof(NoEmojiKeyboardEffect), nameof(NoEmojiKeyboardEffect))]
 namespace Bit.Droid.Effects
 {
     public class NoEmojiKeyboardEffect : PlatformEffect
     {
         protected override void OnAttached()
         {
-            if (Element is Entry && Control is FormsEditText editText)
+            if (Control != null && Control is EditText editText)
```

Comment:
By telling `Control is EditText`, if it's then it's not `null` so the first condition can be removed